google/cloud/bigquery/v2/minimal/internal/job_request_test.cc

```diff
@@ -36,23 +36,19 @@ using ::google::cloud::bigquery_v2_minimal_testing::MakePostQueryRequest;
 using ::google::cloud::testing_util::StatusIs;
 using ::testing::HasSubstr;
 
-auto static const kMin = std::chrono::system_clock::now();
-auto static const kMax = kMin + std::chrono::milliseconds(100);
-auto static const kMinInt =
-    std::chrono::duration_cast<std::chrono::milliseconds>(
-        kMin.time_since_epoch())
-        .count();
-auto static const kMaxInt =
-    std::chrono::duration_cast<std::chrono::milliseconds>(
-        kMax.time_since_epoch())
-        .count();
+auto static const kMinCreationTime = 1111111111111;
+auto static const kMaxCreationTime = 1111111111211;
+std::chrono::system_clock::time_point static const kMinTime{
+    std::chrono::milliseconds{kMinCreationTime}};
+std::chrono::system_clock::time_point static const kMaxTime{
+    std::chrono::milliseconds{kMaxCreationTime}};
```

Comment:
These are reintroducing the dependency on `system_clock`'s epoch.  But `from_time_t()` can be your friend again.
```
auto const kMinTime = std::chrono::system_clock::from_time_t(0) + std::chrono::milliseconds{kMinCreationTimeMs};
```